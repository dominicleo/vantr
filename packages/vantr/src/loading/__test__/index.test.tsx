import * as React from 'react';
import { render } from '@testing-library/react';
import Loading from '../';

describe('Loading', () => {
  it('snapshot', () => {
    const component = render(<Loading />);
    expect(component.asFragment()).toMatchSnapshot();
  });
});
