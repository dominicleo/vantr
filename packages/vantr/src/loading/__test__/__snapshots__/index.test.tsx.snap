// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading snapshot 1`] = `
<DocumentFragment>
  <div
    class="vanr-loading vanr-loading-undefined"
  >
    <span
      class="vanr-loading-spinner-undefined"
    >
      <svg
        class="vanr-loading-circular"
        viewBox="25 25 50 50"
      >
        <circle
          cx="50"
          cy="50"
          fill="none"
          r="20"
        />
      </svg>
    </span>
  </div>
</DocumentFragment>
`;
